packages/web-components/src/option/README.md

```diff
@@ -0,0 +1,139 @@
+| `text`              | public    | `string`                                                                                                       |         |                                                                                                                                                                                      |
+| `value`             | public    | `string`                                                                                                       |         |                                                                                                                                                                                      |
+| `form`              | public    | `HTMLFormElement or null`                                                                                      |         |                                                                                                                                                                                      |
+| `proxy`             |           | <code>new Option(&#96;${this.textContent}&#96;, this.initialValue, this.defaultSelected, this.selected)</code> |         |                                                                                                                                                                                      |
+
+<br />
+
+### **Methods**
+
+<br />
+
+| Name                       | Privacy   | Description                                                         | Parameters                                | Return | Inherited From |
+| -------------------------- | --------- | ------------------------------------------------------------------- | ----------------------------------------- | ------ | -------------- |
+| `checkedChanged`           | public    | Updates the ariaChecked property when the checked property changes. | `prev: boolean or unknown, next: boolean` | `void` |                |
+| `defaultSelectedChanged`   | protected |                                                                     |                                           | `void` |                |
+| `disabledChanged`          | protected |                                                                     | `prev: boolean, next: boolean`            | `void` |                |
+| `selectedAttributeChanged` | protected |                                                                     |                                           | `void` |                |
+| `selectedChanged`          | protected |                                                                     |                                           | `void` |                |
+| `initialValueChanged`      | public    |                                                                     | `previous: string, next: string`          | `void` |                |
```

Comment:
This kind of implies that these exist for the use of consumers...they aren't really intended for that. If we want to enumerate these we should probably be specific about intent.